site/content/docs/5.0/components/dropdowns.md

```diff
@@ -749,7 +749,7 @@ Separate groups of related menu items with a divider.
 Place any freeform text within a dropdown menu with text and use [spacing utilities]({{< docsref "/utilities/spacing" >}}). Note that you'll likely need additional sizing styles to constrain the menu width.
 
 {{< example >}}
-<div class="dropdown-menu p-4 text-body-secondary" style="max-width: 200px;">
+<div class="dropdown-menu p-4 text-body-blue" style="max-width: 200px;">
```

Comment:
In this case, the text is on a blue dropdown background so we can just change it to white to match dropdown item styling.

```suggestion
<div class="dropdown-menu p-4 text-white" style="max-width: 200px;">
```